sdk/communication/Azure.Communication.Chat/tests/Azure.Communication.Chat.Tests.csproj

```diff
@@ -10,6 +10,7 @@
   </ItemGroup>
   <ItemGroup>
     <ProjectReference Include="$(AzureCoreTestFramework)" />
+    <ProjectReference Include="..\..\Azure.Communication.Administration\src\Azure.Communication.Administration.csproj" />
```

Comment:
This reference should be removed. Admin package is empty now and is going to be removed